core/server/services/mega/mega.js

```diff
@@ -377,7 +413,15 @@ async function getEmailMemberRows({emailModel, memberSegment, options}) {
     const knexOptions = _.pick(options, ['transacting', 'forUpdate']);
     const filterOptions = Object.assign({}, knexOptions);
 
-    const recipientFilter = transformEmailRecipientFilter(emailModel.get('recipient_filter'), {errorProperty: 'recipient_filter'});
+    let newsletter;
+    if (labsService.isSet('multipleNewsletters')) {
+        try {
+            newsletter = await emailModel.related('newsletter').fetch();
+        } catch (error) {
+            // Ignore NotFoundError
+        }
```

Comment:
Can you use `const newsletter = await emailModel.related('newsletter').fetch({require: false})` instead here? My understanding is that would [ignore a not-found error](https://bookshelfjs.org/api.html#Model-instance-fetch) automatically, in which case if there's some other error we won't be silently hiding it.

Reply:
neat, I didn't know about this property, thanks 👍